general/solomiia_tymoshchuk/lecture3/_2/BirthDayValidator.java

```diff
@@ -0,0 +1,24 @@
+package general.solomiia_tymoshchuk.lecture3._2;
+
+import static java.lang.Integer.parseInt;
+
+public class BirthDayValidator extends AccountValidator
+{
+
+    public BirthDayValidator(Long birthDay){
+        this.birthDay = birthDay;
+    }
+
+    @Override
+    public boolean isValid(){
+        boolean value;
+        int birthDayConverted = parseInt(String.valueOf(birthDay));
+        if(birthDayConverted > 31) {
```

Comment:
you already have an expression that returns a boolean value, you don't need to use an if statement here:
```
return birthDayConverted >= 1 &&  birthDayConverted <= 31;
```